### ISSUE TITLE
[ACCESSIBILITY] Dynamic changes not announced to screen reader users (Filter/Sort) on Events 

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#4417]
**Describe the bug**
Dynamic changes (filter/sort) are not announced to assistive technology. When the user selects “Sort” parameter, and then the page refreshes to show this, the screen reader user should be notified as well. A good way to do this on this screen would be to Announce “Events page” then land focus on “Filter: Lorem; Sort: Ipsum; Events: ####” text which will let the user know what dynamically changes and what parameters were set with both Filter or Sort. Landing on the parameters is a good practice because otherwise there is an undue burden on the AT user to gesture through more to receive the same information.
![20241014_204526](https://github.com/user-attachments/assets/d7a8e14e-4b02-4e50-958a-3b1e82bf44cd)

[WCAG 4.1.3 Status Messages](https://www.w3.org/TR/WCAG21/#status-messages)
**To Reproduce
Steps to reproduce the behavior:**
1.	Using Android, Open app, logged in as student
2.	Turn on TalkBack
3.	Gesture to All Events
4.	Swipe to “sort” 
5.	Choose “Proximity” or any parameter really

**Expected behavior**
The screen should refresh and the screen reader user would hear something to indicate that the sorting worked.

**Actual behavior**
The screen refreshes with sort parameter selected and results shown, however the screen reader user must gesture through the Back > Events> Personal Information > Notifications > Settings > Filter > Sort > Map> Create> Search just to reach the fact that the sort worked and parameters were applied. 

**Screenshots**

https://github.com/user-attachments/assets/d9ee448a-2f2e-45f8-b70f-addce04f6e54

**Testing Environment & Devices (please complete the following information):**
•	OS: Android Version 13
•	Device: Google Pixel 4a

**Comment by acovher2** [comment]
@TodorBachvarov @pmarkhennessy Due to the ADA title II changes and deadline of April 2026, I wanted to ensure this gets on the list and radar as we need to be compliant. This was on the PPT I sent you.

**Comment by acovher2** [comment]
@TodorBachvarov Thank you for opening #4996. I believe this should be enough as long as focus changes and the filtered data is pronounced. I will await version 7.0.13 to retest this. Thanks.

**Comment by acovher2** [comment]
@TodorBachvarov This is much improved, so close to done. I notice it is reading what it is filtered for but not amount of results. Is there any way to add that? This can let the Screen reader user know in advance how many things they will need to swipe through.

**Comment by acovher2** [comment]
@TodorBachvarov Please let me know about if the screen reader user can have focus land on amount of results first. Thank you

**Comment by TodorBachvarov** [comment]
@acovher2, updated in version 7.2.13. I prefer to avoid moving the Accessibility focus as suggested in the Flutter Accessibility Documentation. Instead I've added additional hint to the Filter/Sort buttons (Result: X events). Please let me know if this brings enough clarity to the user.

**Comment by acovher2** [comment]
Verified as fixed in 7.2.22
